gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Sat Jul 11 01:04:47 CEST 2015
+#Thu Nov 05 21:23:10 EST 2015
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.8-bin.zip
```

Comment:
You could add also `distributionSha256Sum` which was added in Gradle 2.6  - https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:verification